wsd/ClientSession.cpp

```diff
@@ -1170,7 +1170,7 @@ bool ClientSession::handleKitToClientMessage(const char* buffer, const int lengt
         {
             if (errorCommand == "load")
             {
-                LOG_WRN("Document load failed: " << errorKind);
+                LOG_ERR("Document load failed: " << errorKind);
```

Comment:
Careful, this isn't necessarily an error that we want to report! When a document is password protected, loading will fail, and we notify the user and prompt for the password. This isn't an error in the sense that something went wrong.

I think warning here is reasonable. But we can *add* an ERR log in the `else` case, when errorKind is not the three cases we recognize. (That is, add `else LOG_ERR` in line 1197 and leave this one as WRN.)